app/models/ticket.rb

```diff
@@ -18,7 +18,9 @@ def bought?(user)
   end
 
   def paid?(user)
```

Comment:
Where do we use ``paid?``? When you change the method logic you should also check if we still use it correct!